pkgs/top-level/haskell-packages.nix

```diff
@@ -132,8 +130,8 @@ in
         inherit (buildPackages.python311Packages) sphinx; # a distutils issue with 3.12
         python3 = buildPackages.python311; # so that we don't have two of them
         inherit (buildPackages.darwin) autoSignDarwinBinariesHook xattr;
-        buildTargetLlvmPackages = pkgsBuildTarget.llvmPackages_12;
-        llvmPackages = pkgs.llvmPackages_12;
+        buildTargetLlvmPackages = null;
```

Comment:
Is your LLVM removal change ready to merge? If not we could change these to `null` in that PR given that we can technically still support this for the moment.

Reply:
I hope it will merge very imminently. But good point that the LLVM bumps/broken markings can happen in that PR, since they won’t cause rebuilds. Removed from this PR, will add to that one once this is merged (since it touches the same lines).